sdk/nanotdf_test.go

```diff
@@ -934,6 +935,182 @@ func (s *NanoSuite) Test_NanoTDF_Obligations() {
 	}
 }
 
+func (s *NanoSuite) Test_PolicyBinding_GMAC() {
+	// Create test policy data
+	policyData := []byte(`{"body":{"dataAttributes":["https://example.com/attr/classification/value/secret"]}}`)
+
+	// Create GMAC binding - need to simulate having GMAC at end of the digest
+	gmacBytes := []byte{0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08}
+	// Append GMAC to the digest to simulate real scenario
+	policyData = append(policyData, gmacBytes...)
+
+	digest := ocrypto.CalculateSHA256(policyData)
+
+	// For testing, we will use the last bytes as the GMAC binding
+	gmacBytes = digest[len(digest)-len(gmacBytes):]
+
+	binding := &gmacPolicyBinding{
+		binding: gmacBytes,
+		digest:  digest,
+	}
+
+	// Test String function
+	s.Require().Equal(hex.EncodeToString(gmacBytes), binding.String(), "GMAC hash should return binding data directly")
```

Comment:
You can just use the normal `assert` for most scenarios. Generally you should only use the `require` variants when any subsequent assertions or other work won't make sense. Here, we might find that the string encoding is incorrect, but the `Verify` below still works. 

```suggestion
	s.Equal(hex.EncodeToString(gmacBytes), binding.String(), "GMAC hash should return binding data directly")
```

If you broke these into separate sub-tests, this wouldn't matter as much, since you would still run the other logical tests (fenced with comments below) either way (with require or assert)